movie-cards

Eugen-Ko opened the pull request:
оформил карточку. скелет джейэс/тимплейтс в хбс, цсс в мувиекард.цсс. в генреЛист - список жанров. В нормализатионОбж конвертор для нормализации объектов. креатемовиесЛист - там создал временый фетч для получения популар листа, пока Настя класс не напиушет. И там же функция создания листа фильмов - можно использовать для создания листов.

Eugen-Ko committed:
movie-cards

оформил карточку. скелет джейэс/тимплейтс в хбс, цсс в мувиекард.цсс. в генреЛист - список жанров. В нормализатионОбж конвертор для нормализации объектов. креатемовиесЛист - там создал временый фетч для получения популар листа, пока Настя класс не напиушет. И там же функция создания листа фильмов - можно использовать для создания листов.

VladimirBludov reviewed (changes requested):
Удалить лишние файлы

Eugen-Ko committed:
more then 3 genres

Забыл сделать.
Исправил, что если больше 3-х жанров - выводить первые 2а и other..

Eugen-Ko committed:
- цсс

Eugen-Ko committed:
shot genres and titles

изменил место проведения укорачивания, поскольку нормализешрОбж - может использоваться как для листа так и для модалки - а в модалке укорачивания не нужно.

Eugen-Ko committed:
small fix

Eugen-Ko committed:
Update _movieCard.scss

Доделана разметка карточки
